packages/components/badge/src/README.mdx

```diff
@@ -12,8 +12,11 @@ Badge is used to label or highlight items in the interface.
 
 ## Table of contents
 
+- [Table of contents](#table-of-contents)
```

Comment:
My IDE adds this automatically when formatting 😅  I saw this is in other READMEs so I thought it would be fine for now. In the end, someone will align everything at once so I'd keep it for now if you're fine with that.

Reply:
It's only in the input READMEs and the old master README. We can remove before the final merge to the v4 branch 👍

Reply:
I tried to tell my MD plugin (Markdown All In One) to ignore this specific heading or headings before the TOC in general but that's not possible. There are only solutions without disabling the plugin and all its formatting skills:
a) Add ` <!-- omit in toc --> ` after the heading but I guess we don't wanna have that in our code.
b) Exclude it explicitly in the settings.json like this:
```json
"markdown.extension.toc.omittedFromToc": {
    "packages/components/forms/src/checkbox-field/README.mdx": [
      "## Table of contents"
    ],
    "packages/components/forms/src/controlled-input-field/README.mdx": [
      "## Table of contents"
    ],
    "packages/components/forms/src/radio-button-field/README.mdx": [
      "## Table of contents"
    ]
  }
```